### ISSUE TITLE
Should sort by index by default in WAD archives

### ISSUE THREAD

**Issue by Hexagonaldonut** [sirjuddington/SLADE#457]
Funny, a moment ago this was about freakish arbitrary sorting. It isn't anymore, but that's only because I found the 'sort by index' option. (I swear, right-clicking that bar did nothing earlier.)

That said, I probably don't have to say that being able to see and modify the actual order of entries in a WAD archive is _really damn important_, and as of right now SLADE seems to by default sort the entry list with some arcane sorting method that likes to put all the markers at the top. And the indexes of entries are NOT displayed by default (which wouldn't be a big deal if not for the fact that you can't have the list display as it actually is WITHOUT index being visible).

Basically what I'm saying is the index option on the entry list's header should be visible by default (at least in WADs), and WAD archives should auto-sort by index. Unfortunately, simply having the option exist isn't quite enough, as I spent a good few hours futzing with the thing and trying to figure out how to fix all those markers being at the top, before I finally stumbled upon the solution to the problem. And if I spent a lot of time trying to get the thing to work, it's likely other people will too... and not all of them have the level of patience that I do.

...not that I'm that patient, but... you know what I mean.

**Comment by sirjuddington** [comment]
It does default to sorting by index, so I'm not sure what happened if you mean you started SLADE and it immediately wasn't sorting by index. Are you sure you didn't click the 'Size' column at some point, which would sort by size ascending (so you would get all markers at the top)?

**Comment by Hexagonaldonut** [comment]
That would explain a few things, but to my knowledge I haven't ever told it to sort by size, and the size column isn't visible, so I'm not sure what's going on with it.

**Comment by sirjuddington** [comment]
So it still does it when you first start up SLADE?
